src/connectors/meraki.py

```diff
@@ -0,0 +1,181 @@
+"""Meraki
```

Comment:
Let's make this name a little more descriptive -- there are a lot of things we could potentially collect from Meraki, and the titles should  be distinct.